k8s/namespaces/em/kustomization.yaml

```diff
@@ -7,4 +7,5 @@ bases:
 - em-anno/em-anno.yaml
 - em-stitching/em-stitching.yaml
 - em-ccd-orchestrator/em-ccd-orchestrator.yaml
-- em-hrs-api/em-hrs-api.yaml
\ No newline at end of file
+- em-hrs-api/em-hrs-api.yaml
+- em-hrs-ingestor/em-hrs-ingestor.yaml
```

Comment:
this adds you to all environments and will make the app go live? is that intentional?

Reply:
Yes.

Reply:
have you got an ATO / approval to deploy?

Reply:
No. Checking this with my DM